packages/@aws-cdk/aws-chatbot/lib/slack-channel-configuration.ts

```diff
@@ -0,0 +1,283 @@
+          ],
+          resources: ['*'],
+        }),
+      ],
+    }));
+  }
+
+  /**
+   * Allows Lambda-invoke commands in supported clients.
+   */
+  public addLambdaInvokeCommandPermissions(): void {
+    this.configurationRole!.addManagedPolicy(new iam.ManagedPolicy(this, 'LambdaInvokePolicy', {
+      managedPolicyName: 'AWS-Chatbot-LambdaInvoke-Policy',
+      description: 'LambdaInvoke policy for AWS Chatbot',
+      statements: [
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: [
+            'lambda:invokeAsync',
+            'lambda:invokeFunction',
+          ],
+          resources: ['*'],
+        }),
+      ],
+    }));
+  }
+
+  /**
+   * Allows calling AWS Support APIs in supported clients.
+   */
+  public addSupportCommandPermissions(): void {
+    this.configurationRole!.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('AWSSupportAccess'));
+  }
+}
+
+/**
+ * A new Slack channel configuration
+ */
+export class SlackChannelConfiguration extends SlackChannelConfigurationBase {
+
+  /**
+   * Import an existing Slack channel configuration provided an ARN
+   * @param scope The parent creating construct
+   * @param id The construct's name
+   * @param slackChannelConfigurationArn configuration ARN (i.e. arn:aws:chatbot::1234567890:chat-configuration/slack-channel/my-slack)
+   *
+   * @returns a reference to the existing Slack channel configuration
+   */
+  public static fromSlackChannelConfigurationArn(scope: cdk.Construct, id: string, slackChannelConfigurationArn: string): ISlackChannelConfiguration {
+    class Import extends SlackChannelConfigurationBase {
+
+      /**
+       * @attribute
+       */
+      readonly slackChannelConfigurationArn = slackChannelConfigurationArn;
+      readonly configurationRole?: iam.IRole = undefined;
+
+      /**
+       * For example: arn:aws:chatbot::1234567890:chat-configuration/slack-channel/my-slack
+       * The ArnComponents API will return `slack-channel/my-slack`
+       * So i need to handle that to gets a correct name.`my-slack`
+       */
+      readonly configurationName = (() => {
+        const resourceName = cdk
+          .Stack
+          .of(scope)
+          .parseArn(slackChannelConfigurationArn)
+          .resourceName as string;
+
+        return resourceName.replace(/^slack-channel\//, '');
+      })();
+    }
+
+    return new Import(scope, id);
+  }
+
+  readonly slackChannelConfigurationArn: string;
+
+  readonly configurationName: string;
+
+  readonly configurationRole?: iam.IRole;
+
+  constructor(scope: cdk.Construct, id: string, props: SlackChannelConfigurationProps) {
+    super(scope, id, {
+      physicalName: props.slackChannelConfigurationName,
+    });
+
+    this.configurationRole = props.configurationRole || new iam.Role(this, 'ConfigurationRole', {
+      assumedBy: new iam.ServicePrincipal('chatbot.amazonaws.com'),
+    });
+
+    const topicArns = !!props.notificationTopics
```

Comment:
` !!props.notificationTopics` is essentially ` props.notificationTopics`. You can drop double `!`